service/src/test/resources/db/migration/beforeMigrate.sql

```diff
@@ -1,9 +0,0 @@
-/***********************************************************************************************************************
```

Comment:
@jeffjohnson9046 is right and I am wrong.  This file can be used rather than adding a V1 migration script.  I will modify this PR to include that change.